src/Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryFormAbandonEdit.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Flex, ModalDialog, Spacer, Text } from "@artsy/palette"
+import { useRouter } from "System/Router/useRouter"
+
+export const ConsignmentInquiryFormAbandonEditModal: React.FC<{
+  show: boolean
+  onClose: () => void
+}> = ({ show, onClose }) => {
+  const { router } = useRouter()
+
+  if (!show) {
+    return null
+  }
+
+  return (
+    <ModalDialog onClose={onClose} title="Leave without sending message?">
+      <Flex pb={2} flexDirection="column" alignItems={["stretch", "center"]}>
+        <Text>
+          Your message to the Sell with Artsy specialists will not be sent.
+        </Text>
+
+        <Spacer y={2} />
+
+        <Button
+          variant="primaryBlack"
+          onClick={() => {
+            router.go(-2)
```

Comment:
**note:** not sure how we can do this better but I am afraid this is tightly coupled to what we have now, is it possible to do this otherwise or in a more dynamic way

Reply:
I cannot think of anyway to do it to correctly pop to the desired screen without pushing new screen to the stack